### ISSUE TITLE
The amalgamation file includes things like "rapidfuzz/details/Range.hpp"

### ISSUE THREAD

**Issue by oviano** [maxbachmann/rapidfuzz-cpp#104]
Unless I've misunderstood something, they should be embedded into the file, correct?

**Comment by maxbachmann** [comment]
yes this was unintended. I assume this broken includes got added by autocomplete ...
I fixed it in the main branch.
